PayFineControl.java

```diff
@@ -1,65 +1,59 @@
 public class PayFineControl {
 	
-	private PayFineUI Ui;
-	private enum CONTROL_STATE { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
-	private CONTROL_STATE StAtE;
+	private PayFineUI payFineUi;
+	private enum FineControlState { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
+	private FineControlState fineState;
 	
-	private library LiBrArY;
-	private member MeMbEr;
-
+	private library library;
+	private member member;
 
 	public PayFineControl() {
-		this.LiBrArY = LiBrArY.INSTANCE();
-		StAtE = CONTROL_STATE.INITIALISED;
+		this.library = library.instanceLibrary();
+		fineState = FineControlState.INITIALISED;
 	}
 	
-	
-	public void Set_UI(PayFineUI ui) {
-		if (!StAtE.equals(CONTROL_STATE.INITIALISED)) {
-			throw new RuntimeException("PayFineControl: cannot call setUI except in INITIALISED state");
+	public void payFineUi(PayFineUI ui) {

```

Comment:
@GoJoeyGo  payFineUi into setPayFineUi if thats seems okay. Thankyou

Reply:
payFineUi can be changed to setPayFineUi

Reply:
great